packages/relay/src/Environment.js

```diff
@@ -1,13 +1,54 @@
 // @flow
 
 import { GetDeviceLocale } from '@kiwicom/mobile-localization';
-import { Environment, Network, RecordSource, Store } from 'relay-runtime';
+import {
+  Environment,
+  Network,
+  RecordSource,
+  Store,
+  Observable,
+  createOperationSelector,
+} from 'relay-runtime';
+import { AsyncStorage } from 'react-native';
 
 type FetcherResponse = {|
   +data: Object,
   +errors?: $ReadOnlyArray<Object>,
 |};
 
+const ASYNC_STORE_KEY = '@OfflineStore:key';
```

Comment:
Just out of curiosity: is there a reason to add `:key` to `ASYNC_STORE_KEY`?

Reply:
No reason - could be whatever you want. I took it from [the docs](https://facebook.github.io/react-native/docs/asyncstorage.html).